doc/guide_B2C_tenant.md

```diff
@@ -0,0 +1,188 @@
+# Guide for configuring B2C tenant for Energy Origin
+
+## Introduction
+
+This document contains a step-by-step guide on how to configure the ETT B2C production tenant in Azure.
```

Comment:
Maybe describe when this is needed

Reply:
added description: when onboarding new 3rd party client.

Reply:
I can't see this?